forward/pubspec.lock

```diff
@@ -578,5 +602,5 @@ packages:
     source: hosted
     version: "3.1.2"
 sdks:
-  dart: ">=3.2.0-194.0.dev <4.0.0"
-  flutter: ">=3.0.0"
+  dart: ">=3.3.0 <4.0.0"
+  flutter: ">=3.18.0-18.0.pre.54"
```

Comment:
somewhere is `flutter: ">=3.20.0-7.0.pre.48"`. Please make it consistent.

Reply:
Even if I make it consistent, that line will be updated as soon as I run `flutter pub get`. It is automatically set by Flutter.

Reply:
what is the root cause of it?

Reply:
see https://github.com/flutter/flutter/issues/144730